sdks/java/core/src/main/java/org/apache/beam/sdk/coders/NullableCoder.java

```diff
@@ -40,22 +37,14 @@
  *
  * @param <T> the type of the values being transcoded
  */
-public class NullableCoder<T> extends StandardCoder<T> {
+public class NullableCoder<T> extends CustomCoder<T> {
```

Comment:
Ditto, here. Even though `null` is a Java-specific interpretation, it is universal to have a coder that is "either a thing or nothing".